src/python/espressomd/system.pyx

```diff
@@ -294,9 +294,10 @@ cdef class System(object):
         def __get__(self):
             return max_cut_nonbonded
 
-    property lattice_switch:
-        def __get__(self):
-            return lattice_switch
+    IF LB or LB_GPU:
+        property lattice_switch:
```

Comment:
Should this really be exposed? And if so why on the `System` class?

Reply:
No, it shouldn't. You are right.